tests/operators/test_great_expectations.py

```diff
@@ -349,4 +349,4 @@ def test_great_expectations_operator__return_json_dict():
     result = operator.execute(context={})
     logger.info(result)
     assert isinstance(result, dict)
-    assert result["success"]
\ No newline at end of file
+    assert result["_success"] # TODO: Update to "success"
```

Comment:
Did you mean to change this back to success before PR or sometime in the future?

Reply:
Good callout - just updated